packages/common/src/utils/alerting.utils.ts

```diff
@@ -28,9 +28,9 @@ export const addAlert = async (
   title: string,
   message: string,
   severity: Severity,
-  metadata?: unknown,
+  metadata: unknown = null,
```

Comment:
Why did you set this default value? Shouldn't we have the optional parameters in the end of the method?

Reply:
Added a comment in this line: adbc5a8e4dd12f4a90eb5384afeb1caa43848630

I'll refactor the `logger` to be shared between projects, so `logger` will stop being a parameter there, so this default to null is temporary until then

Reply:
I still don't understand the default `null`. We use default values for parameters in the end of the list of parameters, which is not the case.

Reply:
Removed the null default in cae81acc04b833d561559f1f5f8156481057c40d